build/Build.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.1.32414.318
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Build", "Build.csproj", "{B76739F9-CD74-42BE-94AB-40EB74B10C80}"
```

Comment:
Der Pfad der Build.csproj stimmt nicht. Wenn ich die Solution öffne, findet er das Projekt nicht.
Zusätzlich muss auch das Projekt Build.Common.csproj mit in die Solution aufgenommen werden.